### DISCUSSION TITLE
Can I export my notes from trilium as a file?

### DISCUSSION THREAD

**Discussion by Arium3** [TriliumNext/Trilium#2827] [Q&A] [answered]
Can I export my notes from a specific notetree from trilium as a file for backuping or for switching to a new note-app if I need to?
And if I can, how can I do it?

**Comment by XXXJetfireXXX** [comment, marked as answer]
You can right-click the subtree and select "Export"
![image](https://user-images.githubusercontent.com/43646178/166100641-83b33da9-91ce-4843-972a-d9d0d8af6f37.png)
Then you can choose which format do you want to export to:
![image](https://user-images.githubusercontent.com/43646178/166100664-85363f4f-2114-4cc6-b5c3-07964a7e7d15.png)

**Reply by Arium3** [reply]
Thank you so much! This will make manual backups so much easier!
